deploy/helm/templates/metrics-deployment.yaml

```diff
@@ -43,6 +43,8 @@ spec:
                   name: solarwinds-api-token
                   key: SOLARWINDS_API_TOKEN
                   optional: true
+            - name: HTTPS_PROXY
```

Comment:
I believe this is not right place to define this environment variable. It should be in https://github.com/solarwinds/swi-k8s-opentelemetry-collector/blob/master/deploy/helm/templates/common-env-config-map.yaml which is common for Otel collectors for metrics/events/logs, I believe this is applicable to all of them.

Reply:
You're right.